.github/workflows/mysql-migtests.yml

```diff
@@ -60,34 +68,35 @@ jobs:
         psql "postgresql://yugabyte@yb-tserver-n1:5433/yugabyte" -c "SELECT version();"
 
     - name: "TEST: mysql-sakila"
-      run: migtests/scripts/run-test.sh mysql/sakila
+      run: migtests/scripts/run-test.sh mysql/sakila ${{ matrix.use_debezium }}
```

Comment:
Can we set a "job wide" environment variable named USE_DEBEZIUM (using something like https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idenv